README.md

```diff
@@ -59,28 +59,68 @@ the new one and cause confusion).
 
 If the `--host-port` flag is omitted, the emulator will choose a random port,
 which has a high likelihood of not being in use. The emulator will output which
-address to use by displaying a line like `export
-FIRESTORE_EMULATOR_HOST=::1:8195`.
+address to use by displaying a line like `export FIRESTORE_EMULATOR_HOST=::1:8195`.
```

Comment:
Can you restore these? Trying to keep everything wrapped to 80 columns (even though I agree that it looks ugly in this case). Same below.

Reply:
Yeah sorry thats my markdown formatter :)